### ISSUE TITLE
sum totals of grouped values

### ISSUE THREAD

**Issue by GlauberF** [jobsta/reportbro-designer#72]
Without a doubt this package is one of the best I've ever found.
today I have a difficulty in summing a column when there are groupings.
I need to sum the column total, separating by grouping.
I don't know if there is any expression, to get and store the values of each row.
if there was a way to create a variable between the list itself that receives the value of a field and sums it up with the existing one, it would be perfect.
example in the image below.
![Captura de Tela_selecionar área_20220210174114](https://user-images.githubusercontent.com/11314585/153948174-35675565-b54c-4846-b8db-97d0e7242b63.png)

**Comment by alhman** [comment]
table grouping is explained in this guide:
https://www.reportbro.com/guide/view/5/table-grouping

**Comment by GlauberF** [comment]
@alhman yes, I looked at this post, I thought there was another way to do it through the library without having to create a line in the processing with the total values.

**Comment by alhman** [comment]
Sums in table groups are now supported with the latest version 2.1.0

If you create a Parameter of type "Sum" or "Average" and use this parameter in a table group, the value will be calculated based on the current group.

**Comment by GlauberF** [comment]
This functionality is very interesting. Thank you.
would have some example to share, it is easier to understand how to use.
